### PR TITLE
Instead of adding database CRUD, just directly manipulate the new UserSet column.

### PR DESCRIPTION
**Pull request by drgrice1**
The database CRUD was inefficient and overkill.  Basically, its design resulted in an excessive number of database queries for data that has already been obtained from the database.   This approach doesn't need any additional database access when rendering a problem.  The user set already has the necessary data.  Just use it.  When a problem is submitted with this data the data is processed and added to the new column.  This just requires obtaining a clean copy of the user set from the database and then saving the data to the new column.  This is much like the processing of answers for the problem.  Note that the new course environment option for using external data is removed.  With this approach the only time that additional database access is needed is if a problem actually uses external data.

Note that the data for this is now not also stored in the last_answer column for the problem.

The unit test is removed.  It just isn't needed.  That means the dev conf file is also not needed.

### CODE REVIEW

File: lib/WeBWorK/DB.pm [hunk 1]
```diff
@@ -97,14 +97,12 @@ use warnings;
...
 use Scalar::Util qw/blessed/;
-use HTML::Entities qw( encode_entities );
-use Mojo::JSON qw(encode_json decode_json);
-
 use WeBWorK::DB::Schema;
 use WeBWorK::DB::Utils qw/make_vsetID grok_vsetID grok_setID_from_vsetID_sql
 	grok_versionID_from_vsetID_sql/;
 use WeBWorK::Debug;
 use WeBWorK::Utils qw(runtime_use);
+use HTML::Entities qw( encode_entities );
```
Comment (pstaabp) [inline lib/WeBWorK/DB.pm]: I had moved this up because `HTML::Entities` is not a WeBWorK package, but from CPAN. Isn't that how we are organizing these?

Comment (drgrice1) [reply lib/WeBWorK/DB.pm]: I agree.  That is how it should be.  I just reset that file back to the way it was in the develop branch though.

Comment (drgrice1) [reply lib/WeBWorK/DB.pm]: Feel free to add that back.

Comment (drgrice1) [reply lib/WeBWorK/DB.pm]: I went ahead and added it back in this pull request.

**Comment by pstaabp** [comment]
This looks much simpler than I had.  Thanks.

**Pull request merged**